package exceptionHandleDemo;

public class Exception4 {
	
	public static void main(String[] args) {
		
		
		System.out.println("welcome all");
		
		int  a[] = new int[5];
		
		a[4] = 30;
		System.out.println(a[4]);
		
		a[16] = 60;
		System.out.println(a[6]);
		
		
		System.out.println("remaininig code1");
		System.out.println("remaininig code2");
		
		
	}

}
